myapp/spec/system/task_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+
```

Comment:
also according to your task model, you will need a user to create any task.
So, let's create a user here.

```
before do
    create(:user)
 end
```